test/sqllogictest/table_func.slt

```diff
@@ -229,3 +229,19 @@ EXPLAIN PLAN FOR SELECT * FROM x x1, x x2, generate_series(x1.a, x2.a) WHERE x1.
 | Filter (#1 = #3)
 
 EOF
+
+query I
+SELECT * FROM generate_series(0,3), repeat(generate_series);
+----
+1
+2
+2
+3
+3
+3
+
+query I
+SELECT abs(generate_series) FROM generate_series(-1, 2), repeat(generate_series);
+----
+2
+2
```

Comment:
Having blathered a bit about negative numbers, I have to admit not understanding what this is doing!

Reply:
Oh wait, I get it now sorry. The -1 and 1 cancel once you abs them.

Reply:
Step 1: `generate_series(-1, 2)`

```
-1
0
1
2
```

Step 2: `generate_series(-1, 2), repeat(generate_series)`
```
(data, diff)
(-1, -1)
(0, 0)
(1, 1)
(2, 2)
```

Step 3: `SELECT abs(generate_series) FROM generate_series(-1, 2), repeat(generate_series)`
```
(1, -1)
(0, 0)
(1, 1)
(2, 2)
```

Whack that with a consolidate and you get

```
(2, 2)
```

And then expand to unary:

```
2
2
```